pkgs/misc/vim-plugins/update.py

```diff
@@ -310,6 +310,7 @@ def generate_nix(plugins: List[Tuple[str, str, Plugin]]):
                 f"""
   {plugin.normalized_name} = buildVimPluginFrom2Nix {{
     name = "{plugin.normalized_name}-{plugin.version}";
```

Comment:
Would that work?

```suggestion
    version = plugin.version;
```

Reply:
Because `stdenv` is computing name anyway.

Reply:
Not quite as easily, because `buildVimPlugin` does its own name building. But I have pushed a new commit to add that functionality. I've also refactored the `buildVimPlugin` related functions into its own file, since `vim-utils.nix` is doing too much.